client/emv/cmdemv.c

```diff
@@ -753,7 +798,7 @@ int CmdHFEMVExec(const char *cmd) {
 				res = MSCComputeCryptoChecksum(true, (uint8_t *)udol_data_tlv->value, udol_data_tlv->len, buf, sizeof(buf), &len, &sw, tlvRoot);
 				if (res) {
 					PrintAndLog("ERROR Compute Crypto Checksum. APDU error %4x", sw);
-					return 5;
+					dreturn(9);
```

Comment:
Need to free `udol_data_tlv` ?

Reply:
deleted

Reply:
Deleted? What?

Reply:
deleted `udol_data_tlv` declaration and move it to another place

Reply:
found. will fix